src/components/collection/Selector/List/index.tsx

```diff
@@ -102,9 +102,8 @@ function CollectionSelectorList({
     const formStatus = useFormStateStore_status();
 
     const selectionEnabled =
-        currentBindingUUID &&
-        setCurrentBinding &&
-        formStatus !== FormStatus.UPDATING;
+        // currentBindingUUID &&
```

Comment:
This commented out line of code can be removed outright if it is no longer needed.

Reply:
Good catch... this needs added back in. This was a part of testing out not setting a default binding to show.